packages/perspective-viewer-d3fc/src/js/axis/crossAxis.js

```diff
@@ -35,14 +35,22 @@ export const scale = (settings, settingName = "crossValues") => {
 
 const defaultScaleBand = () => minBandwidth(d3.scaleBand());
 
+const flattenArray = array => {

```

Comment:
Could this be added to utils.js as it seems quite generic?

Reply:
I think it probably makes sense to do that if/when we need to reuse it somewhere (though I realise we don't consistently use that logic).

Reply:
Fair enough. I did want to make sure you knew I did actually check the code :-)